paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1245,6 +1245,23 @@ class PaparazziPluginTest {
     )
   }
 
+  @Test
+  fun accessibilityInsets() {
+    val fixtureRoot = File("src/test/projects/accessibility-insets")
+
+    gradleRunner
+      .withArguments("testDebug", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
+
+    val snapshotsDir = File(fixtureRoot, "build/reports/paparazzi/images")
+    val snapshots = snapshotsDir.listFiles()?.sortedBy { it.lastModified() }
+    assertThat(snapshots!!).hasSize(1)
+
+    val image = snapshots[0]
+    val goldenImage = File(fixtureRoot, "src/test/resources/accessibility_insets.png")
+    assertThat(image).isSimilarTo(goldenImage).withDefaultThreshold()
+  }
+
```

Comment:
can we update/reuse the previous test to highlight this?  a new plugin test feels like overkill